extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -46,12 +46,12 @@ export class NetCoreTool {
 		vscode.window.showInformationMessage(NetCoreInstallationConfirmation, UpdateNetCoreLocation, InstallNetCore).then(async (result) => {
 			if (result === UpdateNetCoreLocation) {
 				//open settings
-				vscode.commands.executeCommand('workbench.action.openGlobalSettings');
+				await vscode.commands.executeCommand('workbench.action.openGlobalSettings');
```

Comment:
These are fine to add but aren't going to change anything since it's just in the .then which isn't being awaited itself

Reply:
It should be easier/cleaner to place awaits if you change the model to be async-await rather than chaining Thenables.  Bonus of it being the more modern javascript async model.

Reply:
awaiting Thenables awaits for the .then() from what i remember.

Reply:
Fixed it.